third_party/2and3/characteristic/__init__.pyi

```diff
@@ -0,0 +1,34 @@
+from typing import Sequence, Callable, Union, Any, Optional, AnyStr, TypeVar, Type
+
+def with_repr(attrs: Sequence[Union[AnyStr,Attribute]]) -> Callable[..., Any]: ...
```

Comment:
There should be a space after the comma between `AnyStr` and `Attribute` (here and below).

@ambv -- I'm surprised flake8 didn't catch this?